README.md

```diff
@@ -32,7 +32,7 @@ A number of command-line arguments are available to the parser, some optional, a
 | Option | Description |
 |-|-|
 | `accessToken` | LaunchDarkly [personal access token](https://docs.launchdarkly.com/docs/api-access-tokens) with writer-level access, or access to the `code-references` [custom role](https://docs.launchdarkly.com/v2.0/docs/custom-roles) resource |
-| `dir` | Path to existing checkout of the git repo. If a cloneEndpoint is provided, this option is not required. |
+| `dir` | Path to existing checkout of the git repo. |
```

Comment:
also mentioned in the documentation readme, but this should specify "Absolute path"